frontend/src/pages/AllRunsList.tsx

```diff
@@ -101,7 +104,8 @@ export class AllRunsList extends Page<{ namespace?: string }, AllRunsListState>
 
 const EnhancedAllRunsList = (props: PageProps) => {
   const namespace = React.useContext(NamespaceContext);
-  return <AllRunsList key={namespace} {...props} namespace={namespace} />;
+  const { t } = useTranslation(['experiments', 'common']);
```

Comment:
here both common and experiments translation files are set to t, but only the common part seems to be used by the element. Is the other used by a child? 
Was the purpose to sent t to RunList through the props?